__tests__/app.test.js

```diff
@@ -438,3 +438,26 @@ describe("GET /api/articles (topic query)", () => {
       });
   });
 });
+
+describe("GET :article_id comment_count", () => {
```

Comment:
When we're doing our describe description it's good practice to include the full endpoint! We might use `:article_id` as a parametric input on another endpoint!